packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -840,6 +840,25 @@ const schema: SchemaType<"Posts"> = {
     hidden: !isLWorAF
   },
 
+  glossary: {
+    type: Array,
+    typescriptType: "GlossaryTerm[]",
```

Comment:
Ehm... I think you don't need this (and also this isn't a valid typescript type).

Reply:
(nvm, I see what's going on here.  see my other comments on other related changes.)